ipatests/test_integration/test_commands.py

```diff
@@ -1598,6 +1598,43 @@ def test_ipa_cacert_manage_prune(self):
 
         assert isrgrootx1_nick in result
 
+    def test_ipa_systemd_journal(self):
+        """
+        This testcase checks that administrative user credentials
+        is not leaked to journald log
+        """
+        tasks.kinit_admin(self.master)
+        tasks.kinit_admin(self.replicas[0])
+        tasks.kinit_admin(self.clients[0])
+        cmds = [
+            ['/usr/sbin/ipa-adtrust-install', '-a',
+             self.master.config.admin_password, '-U'],
+            ['/usr/sbin/ipa-replica-manage', 'del',
+             f"dummyhost.{self.master.domain.name}", '-p',
+             self.master.config.dirman_password],
+            ['/usr/sbin/ipa-csreplica-manage', 'del',
+             f"dummyhost.{self.master.domain.name}", '-p',
+             self.master.config.dirman_password],
+            ['/usr/sbin/ipa-kra-install', '-p',
+             self.master.config.dirman_password, '-U']
+        ]
+        for cmd in cmds:
+            self.master.run_command(cmd, raiseonerr=False)
+            tasks.check_journal_does_not_contain_secret(
+                self.master, cmd[0]
+            )
+        for cmd in cmds:
+            self.replicas[0].run_command(cmd, raiseonerr=False)
+            tasks.check_journal_does_not_contain_secret(
+                self.replicas[0], cmd[0]
+            )
+        self.clients[0].run_command(
```

Comment:
You can remove lines 1631-1633 as the call to journalctl is already done in the method check_journal_does_not_contain_secret.